sphinx/source/docs/dev_guide/building.rst

```diff
@@ -211,9 +215,11 @@ not work correctly. The .exe will not be available within the Conda
 environment, which means you will use the version available in the base
 install, if it is available. Instead, you can make sure you use the version
 within the environment by explicitly running the bokeh-server python script
-in the root of the bokeh repository, similar to the following example::
+in the root of the bokeh repository, similar to the following example:
+
+.. code-block:: sh
 
-    python bokeh-server --script path\to\<yourapp>.py
+    $ python bokeh-server --script path\to\<yourapp>.py
```

Comment:
Admittedly, the `$` is all over our docs, we would need to go through and get rid of it I might do that during my upcoming docs work.